io_scene_nif/modules/geometry/vertex/vertex_import.py

```diff
@@ -79,7 +79,7 @@ def map_uv_layer(b_mesh, bf2_index, n_triangles, n_uvco, n_data):
             for n_uv_set in range(len(n_data.uv_sets)):
                 # Set the face UV's for the mesh. The NIF format only supports vertex UV's.
                 # However Blender only allows explicit editing of face  UV's, so load vertex UV's as face UV's
-                uv_layer = str(n_uv_set)
```

Comment:
We could go for the simplest route and just do str(n_uv_set) on the mesh end as well. Just sayin, then we don't need a dedicated function for it.

Reply:
Probably would work in most cases as most it would only ever be one.